src/tss2-tcti/tcti-spi-ftdi.c

```diff
@@ -0,0 +1,204 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2022 Infineon Technologies AG
```

Comment:
Should be 2023

Reply:
Fixed. Updated all copyright notices in the commit.